src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -369,6 +369,8 @@ def _sort_rotational_disks(self):
         ssd = []
         for d in self.args.devices:
             rotating.append(d) if d.rotational else ssd.append(d)
+        if self.args.devices == ssd:
+            return
```

Comment:
Sorry to be nit-picky about this, but I think in a few month time nobody will remember the reason for this comparison.

I think something more explicit would be good here.

```suggestion
        if ssd and not rotational:
            # no need for additional sorting, we'll only deploy standalone on ssds
            return
```